systemvm/patches/debian/config/etc/init.d/cloud-early-config

```diff
@@ -1207,33 +1145,27 @@ setup_secstorage() {
   fi
   setup_apache2 $ETH2_IP
 
+  # Deprecated, should move to Cs Python all of it
+  sed -e "s/<VirtualHost .*:80>/<VirtualHost $ETH2_IP:80>/" \
+    -e "s/<VirtualHost .*:443>/<VirtualHost $ETH2_IP:443>/" \
+    -e "s/Listen .*:80/Listen $ETH2_IP:80/g" \
+    -e "s/Listen .*:443/Listen $ETH2_IP:443/g" \
+    -e "s/NameVirtualHost .*:80/NameVirtualHost $ETH2_IP:80/g" /etc/apache2/vhost.template > /etc/apache2/sites-enabled/vhost-${ETH2_IP}.conf
```

Comment:
It didn't fail, it just didn't happen because that piece of shellscript is exclusively for the SecStore, while VirtualRouter already utilizes the Python bits. 

Fixing that migration is unfortunately out of scope for this PR.

The `NameVirtualHost` directive is now added to `CsApp.py`, tnx 4 testing